dev/bots/test/analyze_test.dart

```diff
@@ -293,4 +294,24 @@ void main() {
       '╚═══════════════════════════════════════════════════════════════════════════════\n'
     );
   });
+
+  test('analyze.dart - verifyMaterialFilesAreUpToDateWithTemplateFiles', () async {
+    String result = await capture(() => verifyMaterialFilesAreUpToDateWithTemplateFiles(
+      testGenDefaultsPath,
+      dartPath,
+    ), shouldHaveErrors: true);
```

Comment:
Just a question for my education. Looks like the `chip_template.dart` has the same content as the `chip.dart` in `dev/bots/test/dev/bots/test/analyze-gen-defaults`, why here `shouldHaveErrors` is true?

Reply:
Test string description is different. I think i didn't make it different enough.

Reply:
Ah chip_template.dart vs chip.dart! Got it!